src/components/annotations/annotation_defaults.js

```diff
@@ -16,114 +15,124 @@ var constants = require('../../plots/cartesian/constants');
 
 var attributes = require('./attributes');
 
+module.exports = function handleAnnotationDefaults(
+  annIn,
+  annOut,
+  fullLayout,
+  opts,
+  itemOpts
+) {
+  opts = opts || {};
+  itemOpts = itemOpts || {};
 
-module.exports = function handleAnnotationDefaults(annIn, annOut, fullLayout, opts, itemOpts) {
-    opts = opts || {};
-    itemOpts = itemOpts || {};
-
-    function coerce(attr, dflt) {
-        return Lib.coerce(annIn, annOut, attributes, attr, dflt);
-    }
-
-    var visible = coerce('visible', !itemOpts.itemIsNotPlainObject);
-    var clickToShow = coerce('clicktoshow');
-
-    if(!(visible || clickToShow)) return annOut;
-
-    coerce('opacity');
-    var bgColor = coerce('bgcolor');
-
-    var borderColor = coerce('bordercolor'),
-        borderOpacity = Color.opacity(borderColor);
+  function coerce(attr, dflt) {
+    return Lib.coerce(annIn, annOut, attributes, attr, dflt);
+  }
 
-    coerce('borderpad');
+  var visible = coerce('visible', !itemOpts.itemIsNotPlainObject);
+  var clickToShow = coerce('clicktoshow');
 
-    var borderWidth = coerce('borderwidth');
-    var showArrow = coerce('showarrow');
+  if (!(visible || clickToShow)) return annOut;
 
-    coerce('text', showArrow ? ' ' : 'new text');
-    coerce('textangle');
-    Lib.coerceFont(coerce, 'font', fullLayout.font);
+  coerce('opacity');
+  var bgColor = coerce('bgcolor');
 
-    coerce('width');
-    coerce('align');
+  var borderColor = coerce('bordercolor'),
+    borderOpacity = Color.opacity(borderColor);
```

Comment:
*(aesthetic comment)* 4-space indents make this kind of variable declaration look neater.